vcs/src/main/java/org/openjdk/skara/vcs/git/GitRepository.java

```diff
@@ -295,6 +295,21 @@ public void clean() throws IOException {
         }
     }
 
+    @Override
+    public void reset(Hash target, boolean hard) throws IOException {
```

Comment:
Did you add a unit test for `reset` in `RepositoryTests`?

Reply:
Nope, will do